src/api/sakApi.ts

```diff
@@ -9,3 +9,17 @@ export async function fetchSakByFnr(fnr: string): Promise<ApiClientResult<Sak>>
 export async function fetchSakBySakId(sakId: string): Promise<ApiClientResult<Sak>> {
     return apiClient({ url: `/saker/${sakId}`, method: 'GET' });
 }
+
+export async function avsluttSøknadsbehandling(arg: {
```

Comment:
Hvorfor har denne blitt flyttet hit? Føles litt rart siden den går mot et endepunkt under `/soknad`

Reply:
John mente at man burde direkte oppdatere tillstanden til saken når man trekker en søknad. Noe som ikke helt passer inn i søknad siden back-end kommer til å returnere sak.

Reply:
Vi muterer jo søknaden, og ikke saken, men _prøver_ å ha som motto at man bør returnere det som gjør det lettest for frontend. Det er like lett for backend å returnere søknaden som å returnerer saken, da må frontenden bare finne søknaden i saken basert på id og erstatte den.

Reply:
Jeg er usikker på hvor den bør ligge da, kanskje du kan hjelpe Ramzi litt med det?

Reply:
Tror @ingara tänker att den borde ligga under `søknadApi` och ikke `sakApi` eftersom endepunktet går mot /soknad?

Reply:
aah okei. flyttet tilbake. Tenkte mer i sammenheng med å måtte oppdatere sak og copy paste all inne på sakApi :)